vpr/src/route/route_timing.h

```diff
@@ -15,6 +15,7 @@ int get_max_pins_per_net();
 
 bool try_timing_driven_route(const t_router_opts& router_opts,
                              const t_analysis_opts& analysis_opts,
+                             const std::vector<t_segment_inf>& segment_inf,
```

Comment:
Why do we need this extra argument? (There are already far to many arguments to these functions.... so it would be nice to avoid making it worse.)